### PR TITLE
Add test-cases for mybatis.check-config-location

### PR DESCRIPTION
**Pull request by kazuki43zoo**
I've added test-cases.
Please review.

### CODE REVIEW

File: mybatis-spring-boot-autoconfigure/src/test/java/org/mybatis/spring/boot/autoconfigure/MybatisAutoConfigurationTest.java [hunk 1]
```diff
@@ -106,6 +107,34 @@ public void testWithConfigFile() {
...
+		assertEquals(1, this.context.getBeanNamesForType(SqlSessionFactory.class).length);
+	}
+
+	@Test
+	public void testWithCheckConfigLocationFileDoesNotExists() {
+		EnvironmentTestUtils.addEnvironment(this.context, "mybatis.config:foo.xml",
+				"mybatis.check-config-location=true");
+		this.context.register(EmbeddedDataSourceConfiguration.class,
+				MybatisAutoConfiguration.class);
+		try {
```
Comment (eddumelendez) [inline mybatis-spring-boot-autoconfigure/src/test/java/org/mybatis/spring/boot/autoconfigure/MybatisAutoConfigurationTest.java]: Use ExpectedException Rule

Comment (kazuki43zoo) [inline mybatis-spring-boot-autoconfigure/src/test/java/org/mybatis/spring/boot/autoconfigure/MybatisAutoConfigurationTest.java]: Oh, OK!

Comment (kazuki43zoo) [inline mybatis-spring-boot-autoconfigure/src/test/java/org/mybatis/spring/boot/autoconfigure/MybatisAutoConfigurationTest.java]: @eddumelendez 

I've modified to use the `ExpectedException`.
Please review again.

**Comment by eddumelendez** [comment]
Merged. Thanks!

**Pull request closed without merging**